File: CS/WinDashboard_LinkedInteractivity/Form1.cs
using DevExpress.DashboardWin;

namespace WinDashboard_LinkedInteractivity
{
    public partial class Form1 : DevExpress.XtraBars.Ribbon.RibbonForm
    {
        public DashboardDesigner DControl { get; set; }
        public Form1()
        {
            InitializeComponent();
            DControl = this.dashboardDesigner1;

            dashboardDesigner1.AllowEditComponentName = true;
            dashboardDesigner1.LoadDashboard("Data\\Dashboard.xml");
        }

        private void btnShowChildForm_ItemClick(object sender, DevExpress.XtraBars.ItemClickEventArgs e)
        {
            Form2 f2 = new Form2(this);
            f2.Show();
        }
    }
}

File: VB/WinDashboard_LinkedInteractivity/Form1.vb
Imports DevExpress.DashboardWin

Namespace WinDashboard_LinkedInteractivity
	Partial Public Class Form1
		Inherits DevExpress.XtraBars.Ribbon.RibbonForm

		Public Property DControl() As DashboardDesigner
		Public Sub New()
			InitializeComponent()
			DControl = Me.dashboardDesigner1

			dashboardDesigner1.AllowEditComponentName = True
			dashboardDesigner1.LoadDashboard("Data\Dashboard.xml")
		End Sub

		Private Sub btnShowChildForm_ItemClick(ByVal sender As Object, ByVal e As DevExpress.XtraBars.ItemClickEventArgs) Handles btnShowChildForm.ItemClick
			Dim f2 As New Form2(Me)
			f2.Show()
		End Sub
	End Class
End Namespace
